src/HonzaBotner/appsettings.CvutFit.json

```diff
@@ -7,7 +7,7 @@
         }
     },
     "CVUT": {
-        "AppBaseUrl": "http://fit-cvut-discord-test.herokuapp.com"
+        "AppBaseUrl": "https://fit-cvut-discord-test.herokuapp.com"
```

Comment:
This change is more likely incorrect one. Our app is in heroku accepting  and responding in http and some heroku higher level functionality manages https